src/urlschemehandler.cpp

```diff
@@ -42,12 +42,19 @@ UrlSchemeHandler::handleContentRequest(QWebEngineUrlRequestJob *request)
             return;
         }
     }
+    bool isRedirect = entry.isRedirect();
     try {
         entry = entry.getFinalEntry();
     } catch (kiwix::NoEntry&) {
         request->fail(QWebEngineUrlRequestJob::UrlNotFound);
         return;
     }
+    if (isRedirect) {
```

Comment:
We only use this variable here. Maybe we could use `entry.isRedirect()` directly.